db/migrate/20230920200529_adding_reference.rb

```diff
@@ -0,0 +1,5 @@
+class AddingReference < ActiveRecord::Migration[7.0]
+  def change
+
+  end
+end
```

Comment:
If the migration was empty why did you created it?

Reply:
It was an error, I know you have to create a new migration if you want to alter your database. But, in this particular case, I ran the command unintentionally.